templates/al2023/runtime/rootfs/etc/systemd/system/nodeadm-run.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=EKS Nodeadm Boot
+Documentation=https://github.com/awslabs/amazon-eks-ami
+# start after cloud-init, in order to pickup changes the
+# user may have applied via cloud-init scripts
+After=nodeadm-config.service cloud-final.service
+Requires=nodeadm-config.service
+
+[Service]
```

Comment:
wondering if we should include a way to disable this, like [ConditionPathExists=](https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html#ConditionPathExists=)?

Reply:
Is there any benefit over `systemctl disable nodeadm-run`?

Reply:
very fair 😅 didn't occur to me when i thought about it

Reply:
Had no idea `ConditionPathExists` was a thing, filing that away for future ref...